From be4f0b0ce92a117631eba25a66b637f99e0d0626 Mon Sep 17 00:00:00 2001
From: Shaveta Leekha <shaveta@freescale.com>
Date: Wed, 3 Dec 2014 12:08:13 +0530
Subject: [PATCH 341/399] drivers/gpio/Kconfig: enable GPIO configs for
 LS1021A SoC

This patch enable configs for GPIOs found on
Freescale's LS1021A SOCs

It also makes mutual exclusion with ARCH_MXC,
since the GPIO controller on LS1020A SOC
is different from ARCH_MXC GPIO controllers

Signed-off-by: Shaveta Leekha <shaveta@freescale.com>
Change-Id: Id5dd8947e2b0f41eb7ea9c3929ce5db8a245b007
Reviewed-on: http://git.am.freescale.net:8181/25120
Tested-by: Review Code-CDREVIEW <CDREVIEW@freescale.com>
Reviewed-by: Jingchang Lu <jingchang.lu@freescale.com>
Reviewed-by: Matthew Weigel <Matthew.Weigel@freescale.com>
[Lu:Original patch taken from
QorIQ-SDK-V1.7-SOURCE-20141218-yocto.iso]
Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 drivers/gpio/Kconfig |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpio/Kconfig b/drivers/gpio/Kconfig
index eba03eb..b2a2fec 100644
--- a/drivers/gpio/Kconfig
+++ b/drivers/gpio/Kconfig
@@ -177,7 +177,7 @@ config GPIO_MPC5200
 config GPIO_MPC8XXX
 	bool "MPC512x/MPC8xxx GPIO support"
 	depends on PPC_MPC512x || PPC_MPC831x || PPC_MPC834x || PPC_MPC837x || \
-		   FSL_SOC_BOOKE || PPC_86xx || ARCH_LAYERSCAPE
+		   FSL_SOC_BOOKE || PPC_86xx || SOC_LS1021A
 	help
 	  Say Y here if you're going to use hardware that connects to the
 	  MPC512x/831x/834x/837x/8572/8610/LS1021A GPIOs.
@@ -207,7 +207,7 @@ config GPIO_MVEBU
 
 config GPIO_MXC
 	def_bool y
-	depends on ARCH_MXC && !ARCH_LAYERSCAPE
+	depends on ARCH_MXC && !SOC_LS1021A
 	select GPIO_GENERIC
 	select GENERIC_IRQ_CHIP
 
-- 
1.7.5.4

